backend/src/main/resources/graphql/main.graphqls

```diff
@@ -237,7 +237,6 @@ type TestOrder {
   symptomOnset: LocalDate
   deviceType: DeviceType
   deviceSpecimenType: DeviceSpecimenType
-  result: String
```

Comment:
learned from @emmastephenson that we can use  `@deprecated(reason: "use results as source of truth")` or something like that here instead of removing them. I can go ahead and clean this up!

Reply:
Done here [65ba33](https://github.com/CDCgov/prime-simplereport/pull/4259/commits/65ba3365f17de52783ccf85b5ddf99b94131f329)